monkey/monkey_island/cc/services/attack/technique_reports/__init__.py

```diff
@@ -10,6 +10,9 @@
 logger = logging.getLogger(__name__)
 
 
+disabled_msg = "This technique has been disabled. You can enable it from the configuration page."
```

Comment:
I'd prefer if this message/this status had a link to the actual config page when presented to the user.

Reply:
@ShayNehmad, found a nicer way to do this than what we had discussed! :)